documentation/src/docs/asciidoc/user-guide/migration-from-junit4.adoc

```diff
@@ -63,6 +63,8 @@ tests to JUnit Jupiter.
 * `@Rule` and `@ClassRule` no longer exist; superseded by `@ExtendWith` and
   `@RegisterExtension`
   - See also <<migrating-from-junit4-rule-support>>.
+* Many assertions and assumptions now accept the message as their last parameter instead of the first one
+  - See also <<migrating-from-junit4-message-parameter, New message parameter position>>.
```

Comment:
```suggestion
  - See <<migrating-from-junit4-failure-message-arguments>> for details.
```

Reply:
Should I omit the name of the anchor link? 
It would be rendered like this:
See [migrating-from-junit4-failure-message-arguments] for details.

Reply:
Hmmm.... it should actually use the heading associated with the link if the link is in the same document.

Reply:
See `(see <<running-tests>> for details` earlier in that document.